### PR TITLE
Fix for redundant naming of tf.json layout files.

### PR DESCRIPTION
**Pull request by Talina06**
Context: Currently the runner appends the suffix .tf.json even when the suffix exists. So, the final files are named: datacenter.tf.json.tf.json.

We want to use override files in the reference layout. https://www.terraform.io/docs/configuration-0-11/override.html

Due to the redundant naming, terraform is unable to identify an *_override.tf.json and the plan fails.

This PR will edit the logic, wherein the suffix ".tf.json" is added only if it is missing.

### CODE REVIEW

File: runner/terraform.go [hunk 1]
```diff
@@ -203,6 +204,10 @@ func (p *Cmd) saveLayout() error {
 		}
 
 		lPath := fmt.Sprintf("%v/%v.tf.json", p.dir, name)
+		if strings.HasSuffix(name, ".tf.json") {
```
Comment (mohandutt134) [inline runner/terraform.go]: Why is this needed?

Comment (Talina06) [reply runner/terraform.go]: @mohandutt134 
Context: Currently the runner appends the suffix .tf.json even when the suffix exists. So, the final files are named: datacenter.tf.json.tf.json.

We want to use override files in the reference layout. https://www.terraform.io/docs/configuration-0-11/override.html

Due to the redundant naming, terraform is unable to identify an *_override.tf.json and the plan fails.

This PR will edit the logic, wherein the suffix ".tf.json" is added only if it is missing.
